openmetadata-ui/src/main/resources/ui/src/components/MetaPilot/MetaPilotProvider/MetaPilotProvider.tsx

```diff
@@ -50,7 +50,8 @@ const MetaPilotProvider = ({ children }: MetaPilotContextProps) => {
   const [entityFqn, setEntityFqn] = useState('');
   const [suggestions, setSuggestions] = useState<Suggestion[]>([]);
   const [loading, setLoading] = useState(false);
-  const [refreshEntity, setRefreshEntity] = useState<() => void>();
+  const [refreshEntity, setRefreshEntity] =
+    useState<(suggestion?: Suggestion) => void>();
```

Comment:
Why we are storing the function in the state? if need so then we can use the ref instead of state.